stdlib/ballerina-builtin/src/main/ballerina/internal/execBallerina.bal

```diff
@@ -0,0 +1,11 @@
+
+public type balCommand "run" | "docker" | "build" | "install" | "uninstall" | "pull" | "push" | "init" | "serach" |
```

Comment:
s/serach/search

Why is the type balComman and not BallerinaCommand?

Reply:
I'll fix this.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/8014